src/woo_publications/metadata/tests/test_organisation_api_endpoints.py

```diff
@@ -0,0 +1,480 @@
+        )
+        organisation2 = OrganisationFactory.create(
+            naam="two",
+            identifier="https://www.example.com/organisaties/2",
+            is_actief=True,
+            oorsprong=OrganisationOrigins.municipality_list,
+        )
+        expected_first_item_data = {
+            "uuid": str(organisation.uuid),
+            "identifier": "https://www.example.com/organisaties/1",
+            "naam": "one",
+            "oorsprong": OrganisationOrigins.custom_entry,
+            "isActief": True,
+        }
+        expected_second_item_data = {
+            "uuid": str(organisation2.uuid),
+            "identifier": "https://www.example.com/organisaties/2",
+            "naam": "two",
+            "oorsprong": OrganisationOrigins.municipality_list,
+            "isActief": True,
+        }
+
+        response = self.client.get(
+            reverse("api:organisation-list"), headers=AUDIT_HEADERS
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.json()
+
+        self.assertEqual(data["count"], 2)
+        self.assertEqual(data["results"][0], expected_first_item_data)
+        self.assertEqual(data["results"][1], expected_second_item_data)
+
+    def test_list_organisations_filter_active(self):
+        organisation = OrganisationFactory.create(
+            naam="one",
+            identifier="https://www.example.com/organisaties/1",
+            is_actief=True,
+            oorsprong=OrganisationOrigins.custom_entry,
+        )
+        organisation2 = OrganisationFactory.create(
+            naam="two",
+            identifier="https://www.example.com/organisaties/2",
+            is_actief=False,
+            oorsprong=OrganisationOrigins.municipality_list,
+        )
+
+        expected_first_item_data = {
+            "uuid": str(organisation.uuid),
+            "identifier": "https://www.example.com/organisaties/1",
+            "naam": "one",
+            "oorsprong": OrganisationOrigins.custom_entry.value,
+            "isActief": True,
+        }
+        expected_second_item_data = {
+            "uuid": str(organisation2.uuid),
+            "identifier": "https://www.example.com/organisaties/2",
+            "naam": "two",
+            "oorsprong": OrganisationOrigins.municipality_list.value,
+            "isActief": False,
+        }
+        list_url = reverse("api:organisation-list")
+
+        with self.subTest("default filter on active true"):
+            response = self.client.get(list_url, headers=AUDIT_HEADERS)
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+            data = response.json()
+
+            self.assertEqual(data["count"], 1)
+            self.assertEqual(data["results"][0], expected_first_item_data)
+
+        with self.subTest("filter on active true"):
+            response = self.client.get(
+                list_url,
+                {"is_actief": OrganisationActive.active},
```

Comment:
the search params are documented as being `camelCase` - please use them that way in the tests too!